app/2.0/docs/upgrade.md

```diff
@@ -787,13 +782,47 @@ this._observer = new Polymer.FlattenedNodesObserver(this._nodesChanged);
 In addition, `Polymer.FlattenedNodesObserver.getFlattenedNodes(node)` can be used to replace the
 `getEffectiveChildNodes` method.
 
+To replace the `getEffectiveChildren` method, use the `getFlattenedNodes` helper method, and filter down to just the elements (ignore comments and text nodes):
+
+1.x {.caption}
+
+```js
+this.getEffectiveChildren
+```
+
+2.x {.caption}
+
+```js
+let effectiveChildren = 
+  Polymer.FlattenedNodesObserver.getFlattenedNodes(this).filter(n.nodeType === Node.ELEMENT_NODE)
+```
+
+To replace the `getContentChildren` method, write platform code to perform this functionality (get the `assignedNodes`, and filter down to just the elements, ignoring comments and text nodes):
+
+1.x {.caption}
+
+```
+this.getContentChildren
```

Comment:
`this.getContentChildren();`

(Or add a variable in both this example and the following one, `var contentChildren =` for 1.x, `let contentChildren =` for 2.x.